src/test/java/com/coyoapp/tinytask/web/TaskControllerTest.java

```diff
@@ -50,6 +45,29 @@ public void shouldCreateTask() throws Exception {
       .andExpect(jsonPath("$.name", is(name)));
   }
 
+  @Test
```

Comment:
I would prefer to extend the existing shouldCreateTask test instead of adding the shouldCreateTaskWithFile (for this testing purpose)